File: gm-utils/js/src/commom/utils/array/ArrayEmpty.js
import IntegerCompare from '../integer/IntegerCompare';
import Null from '../object/Null';

export default class ArrayEmpty {

	static is0(o) {
		return Null.is(o) || IntegerCompare.eq(o.length, 0);
	}

	static is(o) {
		return Null.is(o) || IntegerCompare.eq(o.size(), 0);
	}

}

File: gm-utils/src/main/java/src/commom/utils/array/ArrayEmpty.java
package src.commom.utils.array;

import js.array.Array;
import src.commom.utils.integer.IntegerCompare;
import src.commom.utils.object.Null;

public class ArrayEmpty {

	public static boolean is0(Array<?> o) {
		return Null.is(o) || IntegerCompare.eq(o.lengthArray(), 0);
	}

	public static boolean is(Itens<?> o) {
		return Null.is(o) || IntegerCompare.eq(o.size(), 0);
	}
	
}
